.github/workflows/deploy-front.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Build the image on Cloud Build
         run: |
           chmod +x ./k8s/cloud-build.sh
-          ./k8s/cloud-build.sh front
+          ./k8s/cloud-build.sh front ./front/Dockerfile ../
```

Comment:
Why `../`?

Reply:
Right but for `connectors`, `connectors-edge`, `front-edge` it's `./k8s/cloud-build.sh front ./front/Dockerfile ./`

Reply:
It's an error on my side. I was not able to test this one yet. Fixed.